devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h

```diff
@@ -14,6 +14,7 @@
 #include <unordered_set>
 #include <atomic>
 #include <fstream>
+#include <jsoncpp/json/json.h>
```

Comment:
A preliminary comment. How did you include this dependency? I don't see any modification to the ``CMakeLists``, I am not sure this would be compiling.

Also, consider using ``#include <nlohmann/json.hpp>`` as in https://github.com/ami-iit/bipedal-locomotion-framework/blob/a405c4b96ce6b0265e8fa0556ff47b1d4f1e1791/src/Contacts/src/ContactListJsonParser.cpp#L11C1-L11C29 for example.
This dependency is found in https://github.com/ami-iit/bipedal-locomotion-framework/blob/a405c4b96ce6b0265e8fa0556ff47b1d4f1e1791/cmake/BipedalLocomotionFrameworkDependencies.cmake#L88-L90 and linked in https://github.com/ami-iit/bipedal-locomotion-framework/blob/a405c4b96ce6b0265e8fa0556ff47b1d4f1e1791/src/Contacts/CMakeLists.txt#L16
In this example, notealso that the ``Contact`` module is compiled only if the corresponding ``json`` dependency s found: https://github.com/ami-iit/bipedal-locomotion-framework/blob/a405c4b96ce6b0265e8fa0556ff47b1d4f1e1791/cmake/BipedalLocomotionFrameworkDependencies.cmake#L146-L148